tasks/main.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_list: "{{ capra_lvm_filesystem_packages }}"
+  tags: [capra_lvm_filesystem_prerequisites]
+
+- name: "Configure default mountpoint directory"
+  file:
+    path: "{{ capra_lvm_filesystem_parent_path }}"
+    state: directory
+    owner: "{{ capra_lvm_filesystem_owner }}"
+    group: "{{ capra_lvm_filesystem_group }}"
+    mode: 0755
+    seuser: unconfined_u
+    serole: object_r
+    setype: default_t
+    selevel: s0
+  tags: [capra_lvm_filesystem_prerequisites]
+
+- vars:
+    _capra_lvm_filesystem_path: "{{ item.path | default(capra_lvm_filesystem_parent_path ~ '/' ~ item.name) }}"
+    _capra_lvm_filesystem_state: "{{ item.state | default('present')}}"
+  block:
+    - name: "Unmount volume"
+      mount:
+        path: "{{ _capra_lvm_filesystem_path }}"
+        state: absent
+      with_list: "{{ capra_lvm_filesystem_specs }}"
+      when: _capra_lvm_filesystem_state == 'absent'
```

Comment:
I _really_ dislike mixing two distinct code paths. If you don't want to use `include_tasks` (understandable) you can probably get away with this (still results in static imports):
```
- when: "_capra_lvm_filesystem_state == 'absent'"
  import_tasks: volume-absent.yml

- when: "_capra_lvm_filesystem_state == 'present'"
  import_tasks: volume-present.yml
```

Reply:
It actually added a whole bunch of issues I had to solve, but I did it none the less.